_posts/2021-06-25-gsoc-new-project-first-weeks.md

```diff
@@ -13,47 +13,26 @@ parallax: true #This determines whether or not the asteroid field will appear
 
 World-Gen Modularization Project
 --------------------------------
-This summer, I have begun working on modularizing the World-Gen code in Destination: Sol.
-Currently, Destination: Sol generates the world in the same manner everytime (i.e. the same few Planets, Solar Systems, Mazes, and so on). There's not
-as much variety to the world as there could be, and adding more variety would be difficult with the setup of the code currently.
+This summer, I have begun working on modularizing the World-Gen code in Destination: Sol. Currently, Destination: Sol generates the world in the same manner everytime (i.e. the same few Planets, Solar Systems, Mazes, and so on). There's not as much variety to the world as there could be, and adding more variety would be difficult with the setup of the code currently.
 
-With my project, Destination: Sol will look at World-Gen in a completely new way. Instead of following the same steps everytime to create the world using
-the same few options for how the world should look, the game will follow a general system of how a world will be created. Into that system can be inserted
-any sort of Solar System, Planet, Maze, or anything else a modder might wish to create! Of course, the game world will be generated in the way it
-currently is by default (though it will do this using the new system as well), but with this project the doors will be opened for anyone to add cool, new features.
+With my project, Destination: Sol will look at World-Gen in a completely new way. Instead of following the same steps everytime to create the world using the same few options for how the world should look, the game will follow a general system of how a world will be created. Into that system can be inserted any sort of Solar System, Planet, Maze, or anything else a modder might wish to create! Of course, the game world will be generated in the way it currently is by default (though it will do this using the new system as well), but with this project the doors will be opened for anyone to add cool, new features.
 
-The way in which modders will create their own content is through Java classes called Generators. These will be classes that Destination: Sol will notice
-and be able to use for world generation. The process should be streamlined by the framework of classes that I am setting up, and it will be simple to change
-any aspect of the world. 
+The way in which modders will create their own content is through Java classes called Generators. These will be classes that Destination: Sol will notice and be able to use for world generation. The process should be streamlined by the framework of classes that I am setting up, and it will be simple to change any aspect of the world. 
```

Comment:
I'd maybe say "automatically notice" instead of just "notice", so that it's clearer that they modder doesn't need to take a specific action